modules/container-service/managed-cluster/main.bicep

```diff
@@ -465,7 +515,9 @@ resource managedCluster 'Microsoft.ContainerService/managedClusters@2023-05-02-p
     nodeResourceGroup: nodeResourceGroup
     enablePodSecurityPolicy: enablePodSecurityPolicy
     networkProfile: {
+      networkDataplane: !empty(aksClusterNetworkDataplane) ? any(aksClusterNetworkDataplane) : null
```

Comment:
This comment may affect mulitple places: Should this not be just `networkDataplane` as per the provider? The `aksCluster` in the prefix seems fairly redundant and I assume you added it for consistency reasons as other parameters have it to. So the question is then if we should remove that prefix from all parameters in the template.

Reply:
Love it, thanks